gems/request_store/1.5/manifest.yaml

```diff
@@ -0,0 +1,7 @@
+# manifest.yaml describes dependencies which do not appear in the gemspec.
+# If this gem includes such dependencies, comment-out the following lines and
+# declare the dependencies.
+# If all dependencies appear in the gemspec, you should remove this file.
```

Comment:
Could you remove this file?

Reply:
OK, I deleted in this commit https://github.com/ruby/gem_rbs_collection/pull/239/commits/ba20bf1598b33fad7191824e47c8bca6655595ff